bigchaindb/tendermint/lib.py

```diff
@@ -23,23 +23,28 @@
 
 class BigchainDB(Bigchain):
 
-    def post_transaction(self, transaction):
+    def post_transaction(self, transaction, mode):
         """Submit a valid transaction to the mempool."""
+        mode_list = ('broadcast_tx_async',
```

Comment:
You can move it outside the method, or outside the class, and make it an all capital "constant".